src/main/java/com/sapred/ordermanagerred/service/OrderService.java

```diff
@@ -54,6 +66,50 @@ public List<Order> getOrders(String token, String statusId, int pageNumber, Stri
         return pageOrders.getContent();
     }
 
+    public List<Order> getOrdersByFilters(Map<String, Object> filterMap, String token, int pageNumber) {
+        if (!filterMap.containsKey("companyId")){
```

Comment:
Please don't use filedname as String 
you can add @FieldNameConstants on Order model 
and than you can use Entity.Fields.companyId
this comment is relevant for the method of convert also

Reply:
I have changed it